### ISSUE TITLE
Upload version 1.0.1 to docker hub

### ISSUE THREAD

**Issue by Rez0k** [mongo-express/mongo-express#1170]
Hi, 
First of all, great product.
It will be nice if you will upload version 1.0.1 to docker hub.

Thanks :)

**Comment by rtritto** [comment]
@BlackthornYugen

**Comment by BlackthornYugen** [comment]
@rtritto I don't have permission to publish to docker-hub. I've [created the PR](https://github.com/docker-library/official-images/pull/13162) but that's all I can do.

**Comment by dsaydon90** [comment]
it will be awesome if you can advance this

**Comment by jcpunk** [comment]
I'll confess I'd love to see this get finished

**Comment by GiamBoscaro** [comment]
So, why the releases of the official image are stuck at 1.0.0-alpha4? When are the new images going to get uploaded?

**Comment by mrhackcz** [comment]
Any update? There is 1.0.2 already.

**Comment by BlackthornYugen** [comment]
> Any update? There is 1.0.2 already.

We were able to publish [1.0.0](https://hub.docker.com/_/mongo-express/) a few days ago. But the build wasn't working with 1.0.1 because of an issue we had with yarn. I published 1.0.2 to npm around the same time because I thought it might fix it but we are still stuck at the moment and I'm still not able to put too much time into it since we are not currently using mongo-express where I work. 

Follow the latest over here: https://github.com/mongo-express/mongo-express-docker/issues/97#issuecomment-1745501842
